_includes/integrations/react-quick-start.md

```diff
@@ -55,154 +50,89 @@ This procedure requires:
             onInit={(evt, editor) => editorRef.current = editor}
             initialValue="<p>This is the initial content of the editor.</p>"
             init={% raw %}{{{% endraw %}
-              height: 500,
-              menubar: false,
-              plugins: [
-                'advlist autolink lists link image charmap print preview anchor',
-                'searchreplace visualblocks code fullscreen',
-                'insertdatetime media table paste code help wordcount'
-              ],
-              toolbar: 'undo redo | formatselect | ' +
-              'bold italic backcolor | alignleft aligncenter ' +
-              'alignright alignjustify | bullist numlist outdent indent | ' +
-              'removeformat | help',
-              content_style: 'body { font-family:Helvetica,Arial,sans-serif; font-size:14px }'
+              plugins: 'lists link image table code help wordcount'
             {% raw %}}}{% endraw %}
           />
           <button onClick={log}>Log editor content</button>
         </>
       );
     }
     ```
-    This JavaScript file will create the class `App` containing a {{site.productname}} editor configured to replicate the example on the [Basic example page]({{site.baseurl}}/demo/basic-example/).
 
-5. Provide access to {{site.productname}} using either {{site.cloudname}} or by self-hosting {{site.productname}}.
+    This JavaScript file will create the class `App` containing a {{site.productname}} editor configured with basic features.
```

Comment:
There is no class in the example. I would recommend replacing "class" with "component".

Reply:
```suggestion
    This JavaScript file will create a component "`App`" containing a {{site.productname}} editor configured with basic features.
```